src/ModelContextProtocol/Configuration/McpServerConfig.cs

```diff
@@ -27,11 +27,6 @@ public record McpServerConfig
     /// </summary>
     public string? Location { get; set; }
 
-    /// <summary>
-    /// Arguments (if any) to pass to the executable.
-    /// </summary>
-    public string[]? Arguments { get; init; }
```

Comment:
What's this change for? Is this not actually consumed anywhere?

Reply:
Correct.

> Lastly, I removed the unused McpServerConfig.Arguments property. Arguments are now passed in via McpServerConfig.TransportOptions["arguments"] instead.